.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+extend-ignore =
+    E501
+    E402
```

Comment:
Having googled what E501 actually means, I'd argue that it's a bad idea to ignore it. Instead, it might be better to change the setting that controls its behavior.
It would make sense if you were to use another plugin like [`flake8-length`] to allow occasional low-friction exceptions to it.
FWIW here's a few things I tend to add to this config:
```suggestion
    # E402 -- Module level import not at top of file
    # It is irrelevant for the project because ansible-test enforces boilerplate
    # that violates this rule:
    E402

# Let's not overcomplicate the code:
max-complexity = 10

# Accessibility/large fonts and PEP8 friendly:
# max-line-length = 79
# Accessibility/large fonts and PEP8 unfriendly as set in formatters:
max-line-length = 89

# Count the number of occurrences of each error/warning code and print a report:
statistics = true
```

Some other plugin suggestions include:
- WPS
- darglint
- flake8-2020
- flake8-annotations
- flake8-docstrings
- [`flake8-length`]
- flake8-logging-format
- flake8-pytest-style (I usually add configuration for this one too)
- flake8-spellcheck

[`flake8-length`]: https://pypi.org/p/flake8-length